gnomad/utils/transcript_annotation.py

```diff
@@ -21,61 +21,59 @@
 logger = logging.getLogger("transcript_annotation_utils")
 logger.setLevel(logging.INFO)
 
-TISSUES_TO_EXCLUDE = {
-    "v7": [
-        "Bladder",
-        "Brain_Spinalcord_cervicalc_1",
-        "Brain_Substantianigra",
-        "Cells_EBV_transformedlymphocytes",
-        "Cells_Transformedfibroblasts",
-        "Cervix_Ectocervix",
-        "Cervix_Endocervix",
-        "FallopianTube",
-        "Kidney_Cortex",
-        "MinorSalivaryGland",
-        "Ovary",
-        "Prostate",
-        "Testis",
-        "Uterus",
-        "Vagina",
-    ],
-    "v10": [
-        "Bladder",
-        "Cells_EBV_transformedlymphocytes",
-        "Cells_Culturedfibroblasts",
-        "Cervix_Ectocervix",
-        "Cervix_Endocervix",
-        "Colon_Transverse_MixedCell",
-        "Colon_Transverse_Mucosa",
-        "Colon_Transverse_Muscularis",
-        "FallopianTube",
-        "Kidney_Cortex",
-        "Kidney_Medulla",
-        "Liver_Hepatocyte",
-        "Liver_MixedCell",
-        "Liver_PortalTract",
-        "Ovary",
-        "Pancreas_Acini",
-        "Pancreas_Islets",
-        "Pancreas_MixedCell",
-        "Prostate",
-        "SmallIntestine_TerminalIleum_LymphoidAggregate",
-        "SmallIntestine_TerminalIleum_MixedCell",
-        "Stomach_MixedCell",
-        "Stomach_Mucosa",
-        "Stomach_Muscularis",
-        "Testis",
-        "Uterus",
-        "Vagina",
-    ],
-}
-"""
-List of tissues to exclude from pext analyses and mean pext across tissues. Includes
-reproductive tissues, cell lines, and any tissue with less than 100 samples in the
-specified GTEx version. Tissues in v7 were excluded from gnomAD v2 pext calculations,
-and tissues in v10 were excluded from gnomAD v4 pext. Expression across these tissues
-is still displayed in the gnomAD browser.
-"""
+REPRODUCTIVE_TISSUES = [
+    "FallopianTube",
+    "Ovary",
+    "Prostate",
+    "Testis",
+    "Uterus",
+    "Vagina",
+]
+"""List of reproductive tissues in GTEx."""
+
+CELL_LINES = [
+    "Cells_EBV_transformedlymphocytes",
+    "Cells_Transformedfibroblasts",  # Only in v7.
+    "Cells_Culturedfibroblasts",  # Only in v10.
+]
+"""List of cell lines in GTEx."""
+
+
+def get_tissues_to_exclude(
+    t: Union[hl.Table, hl.MatrixTable],
+    reproductive: bool = True,
+    cell_lines: bool = True,
+    min_samples: Optional[int] = 100,
+) -> List[str]:
+    """
+    Get list of tissues to exclude from pext analyses and mean pext across tissues.
+
+    :param t: Table/MatrixTable with 'tissue' annotation for each sample. If a
+        MatrixTable is input, samples are expected as columns.
+    :param reproductive: Whether to exclude reproductive tissues. Default is True.
+    :param cell_lines: Whether to exclude cell lines. Default is True.
+    :param min_samples: Optional minimum number of samples required for a tissue to be
+        included. If None, tissues will not be excluded based on sample size. Default
+        is False.
```

Comment:
```suggestion
        is 100.
```

Reply:
maybe we should add the paper you found (https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-021-08125-9) to document the reason why we use 100?